certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -888,7 +573,7 @@
               },
               "required": [
                 "BusinessPartnerId",
-                "EnterpriseName",
+                "EnterpriseName1",
```

Comment:
The required input field has been added. Existing flows will not send this new field. EnterpriseName1. Kindly confirm.

Reply:
The previous label was not adequate. For this reason the connector wasn't able to be userd.